haproxy.cfg

```diff
@@ -62,7 +62,8 @@ backend old_domain
 backend percentage_strategy
   http-response set-header X-Proxy-Flow route-to-percentage
   option tcp-check
-  balance leastconn
+  # https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#4.2-balance
+  balance static-rr
```

Comment:
Why don't we use `roundrobin`?

Reply:
i wasnt sure:

> This algorithm is dynamic, which means that server weights may be adjusted on the fly for slow starts for instance.

Reply:
thats static_round_robin which really uses our composed unadjusted weights

Reply:
maybe doesnt matter at all, but leastconn sounds wrong for the percentage strategy